package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "@nestjs/common": "^8.0.7",
-    "@nestjs/core": "^8.0.7",
+    "@nestjs/core": "^9.0.5",
```

Comment:
The latest version of nestJs/core and estjs/common is 9.3.12. Maybe we should use it.